docker/docker-compose.prod.yml

```diff
@@ -100,6 +100,7 @@ services:
     environment:
       - MONGODB_PASSWORD=<<DOCKER-SECRET:EVALUATING_STUDENTS_DB_PASSWORD>>
       - RABBITMQ_PASS=<<DOCKER-SECRET:RABBIT_MQ_PASSWORD>>
+      - SWAGGER_BASE_URL=avantysuniversity.nl/api/v1/evaluating_students
```

Comment:
Wat als je in de testserver zit? Of een andere host

Moet dit er expliciet in? Want wat voegt dit precies toe aan swagger?

Reply:
Tip: als deze optie écht nodig is, werp een blik op mijn Nginx environment variabelen in de hoofd compose